numpy/add_newdocs.py

```diff
@@ -5844,9 +5844,12 @@ def luf(lamdaexpr, *args, **kwargs):
 
         .. versionadded:: 1.7.0
     initializer : scalar, optional
-        The initializer before starting the reduction. The reduction starts
-        with this value rather than the identity. This can also be used to calculate
-        reductions when the slice is empty.
+        The value to which the result is initialized before starting the reduction.
+        Defaults to the `~numpy.ufunc.identity` of the ufunc.
+        If ``None`` is given, then the first element of the reduction is used,
+        and an error is thrown if the reduction is empty. If ``None`` is given to a
+        reduction for a empty axes, it will raise an error. If ``a.dtype`` is ``object``,
```

Comment:
Aren't these two sentences the same?

>  If ``None`` is given, then the first element of the reduction is used, and an error is thrown if the reduction is empty.

and

> If ``None`` is given to a reduction for a empty axes, it will raise an error.

Reply:
Fixed!